### ISSUE TITLE
bug: The supplied auth credential is malformed or has expired. 

### ISSUE THREAD

**Issue by Sampath-Lokuge** [robingenz/capacitor-firebase#74]
Hi,

I use [this Capacitor plugin](https://github.com/robingenz/capacitor-firebase/blob/main/packages/authentication/docs/setup-microsoft.md) to Auth with MS. But no luck yet.

I have followed [this doc](https://firebase.google.com/docs/auth/android/microsoft-oauth#before_you_begin) and configured it all well.

But can you tell me where to set this up?

> When registering apps with these providers, be sure to register the *.firebaseapp.com domain for your project as the redirect domain for your app.

Error

`The supplied auth credential is malformed or has expired. [ Error getting access token from microsoft.com, OAuth2 redirect uri is: https://my-app-123a2.firebaseapp.com/__/auth/handler, response: OAuth2TokenResponse{params: error=invalid_reques`

This is an Ionic/Angular Native app on an Android device.

**Comment by robingenz** [comment]
Hi @Sampath-Lokuge,
you need to register a Microsoft OAuth client in Azure Portal:
![grafik](https://user-images.githubusercontent.com/13857929/164884602-c79485a7-00ff-4a50-8747-6092d30c9188.png)

There you need to provide the redirection URL that will be given to you in the Firebase Console.
![grafik](https://user-images.githubusercontent.com/13857929/164884560-a73816cb-afeb-4f0b-94d7-ab6d91af14bd.png)

**Comment by robingenz** [comment]
![grafik](https://user-images.githubusercontent.com/13857929/164884897-91c8117f-785a-4613-8e83-72a55b2bd399.png)

Let me know if the error message still appears.

**Comment by Sampath-Lokuge** [comment]
Hi @robingenz 

Yes, may I know where to give `https://my-app-12we34.firebaseapp.com/__/auth/handler` on the Azure portal? Thanks!

**Comment by robingenz** [comment]
See my [last comment](https://github.com/robingenz/capacitor-firebase/issues/74#issuecomment-1107412026) (just added 😄 )

**Comment by Sampath-Lokuge** [comment]
Hi @robingenz 

Ok sure. All are working fine now. 

I have realized that this is happening through the Web after your images. i.e. not through the Native layer. But I have spent over 4 H doing that through Native layer configurations on the Azure portal. So It would be great if you'll put this bug/clarification issue link under [here](https://github.com/robingenz/capacitor-firebase/blob/main/packages/authentication/docs/setup-microsoft.m) so as to this will not happen to someone in the future.

Thanks a lot for your great support.
